pkg/osquery/interactive/interactive_test.go

```diff
@@ -0,0 +1,102 @@
+//go:build !windows
+// +build !windows
```

Comment:
I realized that you're using a socket, which won't work on windows. Windows probably needs a named pipe. This is in `pkg/osquery/runtime/runtime_helpers_windows.go`

I think it would be fine to ship as is, and then add windows support